packages/block-editor/src/components/block-toolbar/style.scss

```
@@ -183,6 +183,25 @@
 	}
 }
 
+.block-editor-block-toolbar__block-parent-wrapper {
+	position: absolute;
+	top: -1px;
+	left: -1px;
+	opacity: 0;
+	transition: all 60ms linear;
+	z-index: -1; // This makes it slide out from underneath the toolbar.
+
+	.is-showing-movers & {
+		opacity: 1;
+		transform: translateY(-54px);
```

I think this change will apply the spacing @jasmussen wants, while also avoiding hardcoded values:
```suggestion
		transform: translateY(-($block-toolbar-height + $grid-unit-15));
```

---

Ah, thanks. I added this.